helm-chart/renku-notebooks/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: '1.0'
 description: A Helm chart for the Renku Notebooks service
 name: renku-notebooks
-version: 0.2.0
+version: 0.0-unclean-191b684
```

Comment:
:thinking:

Reply:
Yes I know, I'll clean all this up when I'm done running it on my minikube :)